packages/react/README.md

```diff
@@ -0,0 +1,86 @@
+This package provides [React.js](https://react.dev/) wrappers for [Synergy Web Components](https://github.com/SickDesignSystem/synergy/tree/main/packages/components).
+
+## Getting started
+
+### 1. Package installation
+
+Run the following steps to install the required packages.
+
+```bash
+# Install the base library and required css files
+npm install --save @synergy-design-system/react @synergy-design-system/tokens
+```
+
+> ⚠️ Note we do **not** ship react in this package.
+> You will have to install React by yourself first!
+
+### 2. Add the wanted theme to your application
+
+The components will not display correctly without the needed theme. Please include either light or dark theme in your application, for example in a newly installed vite React application:
+
+```tsx
+// main.tsx
+import { StrictMode } from 'react';
+import { createRoot } from 'react-dom/client';
+import { App } from './App';
+
+// Add this line to enable the light theme for your application
+import '@synergy-design-system/tokens/dist/css/light.css';
+
+createRoot(document.getElementById('root')!).render(
+  <StrictMode>
+    <App />
+  </StrictMode>,
+);
+```
+
+### 3. Importing and rendering components
+
+You may now use the components by importing them from the `@synergy-design-system/react` package and rendering them in a react component.
+
+```tsx
+import { SynButton } from '../dist';
+
+export const MyButton = () => (
+  <SynButton type="button" variant="primary">
+    SynButton Example
+  </SynButton>
+);
+```
+
+## 4. Using component events
+
+This library makes use of [@lit/react](https://lit.dev/docs/frameworks/react/) to wrap the existing Synergy Web Components.
+All events will be automatically set up to work without the need to attach event listeners manually.
+Just use them with the default react `onEVENT` prefix, where `EVENT` is the camelCased name of the event.
+
+```tsx
+import { SynButton } from '../dist';
+
+export const MyButton = () => (
+  <SynButton
+    onSynBlur={e => console.log('button blur event', e)}
+    onSynFocus={e => console.log('button focus event', e)}
+    onSynInvalid={e => console.log('button flagged as invalid', e)}
+  >
+    SynButton Example
+  </SynButton>
+);
+```
+
+---
+
+## Development
+
+To create a new version of this package, proceed in the following way:
+
+1. Check out the [Synergy Design System Repository](https://github.com/SickDesignSystem/synergy).
+2. Run `pnpm i -r` to install all dependencies.
+3. Build the `@synergy-design-system/components` package (or run `pnpm build in the project root to build everything).
+4. Move to to `packages/react` and use `pnpm start` to spin up a local vite project using react and typescript to validate your changes.
+5. Apply all fixes displayed when using `pnpm lint`.
+
+> ⚠️ The build process will always try to sync this packages `package.json.version` field with the latest version from `@synergy-design-system/components`!
+> Therefore, it is best to not alter the version string
```

Comment:
Is this still valid after your changes?

Reply:
No, I am currently in the process of removing all other stuff in the react folder so we get a real auto-generation. The build information will get removed, but we need to have a README.md at least.

Reply:
Plese have a look at the latest version of the readme. Builds are now always only triggered via `pnpm build` and the demo was moved to `_private/react-demo`, where it still works fine.